apps/test/unit/code-studio/pd/professional_learning_landing/EnrolledWorkshopsTest.jsx

```diff
@@ -22,7 +22,9 @@ describe('EnrolledWorkshops', () => {
       facilitators: [],
       organizer: {name: 'organizer_name', email: 'organizer_email'},
       enrollment_code: 'code1',
-      state: 'Not Started'
+      state: 'Not Started',
+      user_id: 123,
+      attended: false
```

Comment:
Not required right now, but seeing this large [fixture](https://stackoverflow.com/questions/12071344/what-are-fixtures-in-programming) again, and how many places you had to edit it, reminded me that we have a better way to generate test data now: [Rosie.js](https://github.com/rosiejs/rosie), the JavaScript equivalent to [FactoryBot](https://github.com/thoughtbot/factory_bot).  [Here's an example](https://github.com/code-dot-org/code-dot-org/pull/30053) of cleaning up a large test fixture with Rosie.